### PR TITLE
ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits: Whitelist passing libvirt tests

### PR DESCRIPTION
**Pull request by wking**
As of now not all parallel tests pass on libvirt like they do on AWS.  With this commit we whitelist known-good tests so we can start gating on libvirt success.  As libvirt support improves, we can expand the list until we have full coverage.

This is an alternative to #3517 which carries the whitelist in local version control, vs. pulling from an external source.  CC @praveenkumar

### CODE REVIEW

File: ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml [hunk 1]
```diff
@@ -186,7 +186,139 @@ presubmits:
         - name: JOB_NAME_SAFE
           value: e2e-libvirt
         - name: TEST_COMMAND
-          value: TEST_SUITE=openshift/conformance/parallel run-tests
+          value: |
```
Comment (abhinavdahiya) [inline ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml]: :scream:

Comment (wking) [reply ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml]: > :scream:

I know, right?  Hopefully we get the full parallel suite passing soon.

**Comment by wking** [comment]
```console
$ curl -s https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_re
lease/3550/rehearse-3550-pull-ci-openshift-installer-master-e2e-libvirt/3/artifacts/e2e-libvirt/container-logs/setup.log.gz | gunzip
/bin/sh: line 147: run-tests: command not found
```

Wait, why does `setup` care about `run-tests`?

**Comment by wking** [comment]
> Wait, why does `setup` care about `run-tests`?

Because I was nesting one `EOF`-delimited here-document (from the job) inside another (from the template).  I've pushed a9716248c -> 95c33436b to use `EOTESTS` for the job's here-document.

**Comment by praveenkumar** [comment]
/lgtm

libvirt test case also now passing.

**Comment by wking** [comment]
> libvirt test case also now passing.

I think you may have jumped the gun.  Are there any JUnit under [here][1]?  I'd have expected some after:

```console
$ curl -s https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/3550/rehearse-3550-pull-ci-openshift-installer-master-e2e-libvirt/4/artifacts/e2e-libvirt/container-logs/test.log.gz | gunzip | tail -n4
129 pass, 0 skip (10m16s)
Writing JUnit report to /tmp/artifacts/junit/junit_e2e_20190423-060319.xml

+ touch /home/packer/exit
```

Can you file a follow-up which collects that `junit` directory?

Also, this doesn't look great:

```console
$ curl -s https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/3550/rehearse-3550-pull-ci-openshift-installer-master-e2e-libvirt/4/artifacts/e2e-libvirt/container-logs/setup.log.gz | gunzip | grep fatal
level=fatal msg="failed to initialize the cluster: Working towards 0.0.1-2019-04-23-045045: 96% complete: timed out waiting for the condition"
```

Can you work on `setup` so it fails if `create cluster` fails?

[1]: https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_release/3550/rehearse-3550-pull-ci-openshift-installer-master-e2e-libvirt/4/artifacts/e2e-libvirt/

**Comment by praveenkumar** [comment]
@wking I will create issue for Junit and work on it soon.

**Pull request merged**